PDSA_System/Server/DataBase/Create.sql

```diff
@@ -17,8 +17,10 @@ CREATE TABLE IF NOT EXISTS Bruker(
 
 CREATE TABLE IF NOT EXISTS Team(
   TeamId INTEGER NOT NULL AUTO_INCREMENT,
+  TeamLederId INTEGER NOT NULL,
   Navn VARCHAR(50),
   AvdelingId INTEGER,
+  FOREIGN KEY (TeamLederId) REFERENCES Bruker(AnsattNr),
   CONSTRAINT PK_Team PRIMARY KEY (TeamId)
 );
```

Comment:
```suggestion
FOREIGN KEY (AvdelingId) REFERENCES Team(TeamId)
);
```
Da er det en unary tilkobling slik vi hadde tenkt. Kan sammenlignes med koblingen mellom bruker og rolle i Bruker-tabellen